main/ZgatewayRTL_433.ino

```diff
@@ -49,8 +44,9 @@ void rtl_433_Callback(char* message) {
     return;
   }
 
+  unsigned long MQTTvalue = (int)RFrtl_433_ESPdata["id"] + round(RFrtl_433_ESPdata["temperature_C"]);
   String topic = String(subjectRTL_433toMQTT);
-#  if valueAsATopic
+#  ifdef valueAsATopic
```

Comment:
Hi @NorthernMan54,

we were trying to get away from the sometimes confusing 'just defined' to go for a correct true/false with https://github.com/1technophile/OpenMQTTGateway/pull/1260, so here it should remain **if**.

Thanks